rareserverapi/views/auth.py

```diff
@@ -34,7 +34,6 @@ def register_user(request):
     rareUser = RareUsers.objects.create(
         bio=req_body['bio'],
         profile_image_url=req_body['profile_image_url'],
-        active=True,
```

Comment:
Since you've removed this from here, you should, up above in the `new_user` variable, set `is_active=True`, since new users should always be created as active users. :)